app/javascript/dashboard/components/widgets/conversation/OnboardingView.vue

```diff
@@ -10,7 +10,20 @@ export default {
     };
   },
   computed: {
-    ...mapGetters({ globalConfig: 'globalConfig/get' }),
+    ...mapGetters({
+      globalConfig: 'globalConfig/get',
+      currentUser: 'getCurrentUser',
+    }),
+    greeting() {
+      const time = new Date();
+      const hours = time.getHours();
+      if (hours < 12) return 'Good morning';
+      if (hours < 18) return 'Good afternoon';
+      return 'Good evening';
+    },
+    greetingMessage() {
+      return `👋 ${this.greeting}, ${this.currentUser.name}. Welcome to ${this.globalConfig.installationName}.`;
+    },
```

Comment:
Use i18n. Read the guidelines for translation strings. Do not break the strings. https://chatwoot.help/hc/handbook/articles/1710791123-translation-guidelines